src/bscPlugin/validation/ScopeValidator.spec.ts

```diff
@@ -2004,6 +2004,43 @@ describe('ScopeValidator', () => {
                 DiagnosticMessages.cannotFindName('outer')
             ]);
         });
+
+        it('allows method call on hex literal', () => {
+            program.setFile('source/main.bs', `
+                function test()
+                    x = &HFF.toStr()
+                    return x
+                end function
+            `);
+
+            program.validate();
+            expectZeroDiagnostics(program);
+        });
+
+
+        it('allows method call on hex literal', () => {
+            program.setFile('source/main.bs', `
+                function test()
+                    x = &HFF.toStr()
+                    return x
+                end function
+            `);
+
+            program.validate();
+            expectZeroDiagnostics(program);
+        });
+
+        it('has no validation errors with print statement with hex followed by dot <number>', () => {
```

Comment:
@markwpearce I was hoping to see a test or two that showed diagnostics for this type of syntax error.

![image](https://github.com/user-attachments/assets/ff516205-7759-4f1e-8da6-a31a0419e7cc)